### PR TITLE
Task3 navigation menu

### PR DESCRIPTION
**Pull request by Platton96**

### CODE REVIEW

File: MoneySplitter.Infrastructure/INavigationManager.cs [hunk 1]
```diff
@@ -1,9 +1,12 @@
-﻿namespace MoneySplitter.Infrastructure
+﻿using Caliburn.Micro;
```
Comment (Vovanella95) [inline MoneySplitter.Infrastructure/INavigationManager.cs]: Do you need this reference here?

File: MoneySplitter.Services/API/SessionApiService.cs [hunk 2]
```diff
@@ -32,18 +32,18 @@ public SessionApiService(IApiUrlBuilder urlBuilder, IQueryApiService queryApiSer
 
             var dataUser = await _queryApiService.PostQueryAsync<DataUser, LoginModel>(loginModel, signInUri);
 
-            return _maper.DataUserToConvertUserModel(dataUser);
+            return _maper.ConvertDataUserToUserModel(dataUser);
```
Comment (Vovanella95) [inline MoneySplitter.Services/API/SessionApiService.cs]: better rename this method to ConvertUserDataToUserModel

File: MoneySplitter.Services/API/SessionApiService.cs [hunk 3]
```diff
@@ -32,18 +32,18 @@ public SessionApiService(IApiUrlBuilder urlBuilder, IQueryApiService queryApiSer
...
         }
 
         public async Task<UserModel> RegistrAsync(RegisterModel registrModel)
         {
             var regisrUri = _urlBuilder.Registration();
 
-            var dataRegisterUser = _maper.RegisterModelToConverDataRegisterModel(registrModel);
+            var dataRegisterUser = _maper.ConvertRegisterModelToDataRegisteUser(registrModel);
```
Comment (Vovanella95) [inline MoneySplitter.Services/API/SessionApiService.cs]: registErModel

Comment (Vovanella95) [inline MoneySplitter.Services/API/SessionApiService.cs]: ConvertRegisterModelToDataRegisteUser
what? ))) RegisteUser?

File: MoneySplitter.Win10/Views/ShellView.xaml.cs [hunk 4]
```diff
@@ -14,9 +14,9 @@ public ShellView()
             DataContextChanged += (s, e) => { ViewModel = DataContext as ShellViewModel; };
         }
 
-        private void ShellFrame_Loaded(object sender, Windows.UI.Xaml.RoutedEventArgs e)
-        {
-            ViewModel.InitializeShellNavigationService(ShellFrame);
-        }
+        //private void ShellFrame_Loaded(object sender, Windows.UI.Xaml.RoutedEventArgs e)
```
Comment (Vovanella95) [inline MoneySplitter.Win10/Views/ShellView.xaml.cs]: Don't leave commented code please

File: MoneySplitter.Infrastructure/INavigationManager.cs (hunk 1 above)
Comment (Platton96) [reply MoneySplitter.Infrastructure/INavigationManager.cs]: fix

File: MoneySplitter.Services/API/SessionApiService.cs (hunk 2 above)
Comment (Platton96) [reply MoneySplitter.Services/API/SessionApiService.cs]: fixed

File: MoneySplitter.Services/API/SessionApiService.cs (hunk 3 above)
Comment (Platton96) [reply MoneySplitter.Services/API/SessionApiService.cs]: rename

File: MoneySplitter.Infrastructure/INavigationManager.cs [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
+namespace MoneySplitter.Infrastructure
 {
     public interface INavigationManager
     {
         void NavigateToShellView();
         void NavigateToRegistrViewModel();
-        //void InitializeShellNavigationService(INavigationService navigationService);
+        void InitializeShellNavigationService(object navigationService);
+        void NavigateToShellViewModel();
+        void NavigateToFriendsViewModelFrame();
```
Comment (Vovanella95) [inline MoneySplitter.Infrastructure/INavigationManager.cs]: Why Frame? And don't use ViewModel word. Just name it NavigateToFriends, NavigateToShell.....

File: MoneySplitter.Win10/ViewModels/LoginViewModel.cs [hunk 6]
```diff
@@ -10,8 +9,8 @@ public class LoginViewModel:Screen
         private readonly INavigationManager _navigationManager;
         private IMembershipService _membershipService;
 
-        private string _email;
-        private string _password;
+        private string _email="q@mail.ru"; //для входа, потом убиру
```
Comment (Vovanella95) [inline MoneySplitter.Win10/ViewModels/LoginViewModel.cs]: Make this fields const

Comment (Vovanella95) [inline MoneySplitter.Win10/ViewModels/LoginViewModel.cs]: Use comments in english

File: MoneySplitter.Win10/Views/RegisterView.xaml.cs [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+        public RegistrView()
+        {
+            InitializeComponent();
+            DataContextChanged += (s, e) => { ViewModel = DataContext as RegisterViewModel; };
+        }
+
+        private async void OnRegistrButtonClick(object sender, RoutedEventArgs e)
+        {
+            await ViewModel.Registred();
```
Comment (Vovanella95) [inline MoneySplitter.Win10/Views/RegisterView.xaml.cs]: What is Registred?

File: MoneySplitter.Win10/Views/ShellView.xaml.cs [hunk 8]
```diff
@@ -14,9 +14,14 @@ public ShellView()
             DataContextChanged += (s, e) => { ViewModel = DataContext as ShellViewModel; };
         }
 
-        private void ShellFrame_Loaded(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        private void OnMenuItemClick(NavigationView sender, NavigationViewItemInvokedEventArgs args)
+        {
+            ViewModel.NavigateToClikedItemMenu((string)args.InvokedItem);
+        }
+
+        private void ShellFrame_Loaded(object sender, RoutedEventArgs e)
```
Comment (Vovanella95) [inline MoneySplitter.Win10/Views/ShellView.xaml.cs]: Rename this method to OnShellFrameLoaded

Comment (Platton96) [reply MoneySplitter.Win10/Views/ShellView.xaml.cs]: rename

File: MoneySplitter.Win10/Views/RegisterView.xaml.cs (hunk 7 above)
Comment (Platton96) [reply MoneySplitter.Win10/Views/RegisterView.xaml.cs]: rename

File: MoneySplitter.Infrastructure/INavigationManager.cs (hunk 5 above)
Comment (Platton96) [reply MoneySplitter.Infrastructure/INavigationManager.cs]: rename

File: MoneySplitter.Win10/ViewModels/LoginViewModel.cs (hunk 6 above)
Comment (Platton96) [reply MoneySplitter.Win10/ViewModels/LoginViewModel.cs]: rename

**Pull request merged**